src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractTests.cs

```diff
@@ -15,7 +16,11 @@ public static void MessageContract_IsWrapped_True()
     {
         StringBuilder errorBuilder = new StringBuilder();
 
-        XmlDictionaryReader reader = MessageContractHelpers.SetupMessageContractTests(isWrapped: true);
+        MessageContractHelpers.IMessageContract clientProxy;
```

Comment:
It would be good to get rid of errorBuilder and to use the try/finally pattern other tests use to close the proxy and factory. The helper makes that slightly more difficult.